_data/scopes.yaml

```diff
@@ -13,6 +13,7 @@ integrations:
     transform-service: "Transform Service"
     transformation-engine: "Document Transformation Engine"
     media-management: "Media Management"
+    search-enterprise: "Search Enterprise"
```

Comment:
Thanks for doing this one. Is it worth our while to put the Search products in the same order they appear in on the docs landing page? Other than that this LGTM.

Reply:
Yes, but it can be done in a future PR for Search Enterprise.